Yubico.YubiKey/src/Yubico/YubiKey/FirmwareVersion.cs

```diff
@@ -40,6 +40,7 @@ public class FirmwareVersion : IComparable<FirmwareVersion>, IComparable, IEquat
         internal static readonly FirmwareVersion V5_3_0 = new FirmwareVersion(5, 3, 0);
         internal static readonly FirmwareVersion V5_4_2 = new FirmwareVersion(5, 4, 2);
         internal static readonly FirmwareVersion V5_4_3 = new FirmwareVersion(5, 4, 3);
+        internal static readonly FirmwareVersion V5_6_0 = new FirmwareVersion(5, 6, 0);
```

Comment:
Should we add 5.7.0 to this list right away?

Reply:
Not in this PR 😄 I would say add it when you need it.

Reply:
Ok, I just did :)